### DISCUSSION TITLE
Saving with Ctrl-S

### DISCUSSION THREAD

**Discussion by genkigenki** [sbrin/lopaka#82] [Q&A] [answered]
This is really great! 
Ctrl-S keeps the "Save" button active. Clicking save however removes the button until changes are done.
1) Is Ctrl-S the same as the save button?
2) Could you just add 5 more workspaces of the same library to switch over? That way, its at least possible to have several GUIs to work with. Would be a quick fix maybe and help a lot.

**Comment by sbrin** [comment, marked as answer]
1. Yes it's the same. The button shouldn't stay after Ctrl+S, it's a bug.
2. I've sent you an access to cloud beta which has this feature, please check your email

**Reply by genkigenki** [reply]
Thanks a lot for the quick reply! 
 I tried your beta, sent you a mail as well (seems localhost is referred to)
